demos/common/python/openvino/model_zoo/model_api/models/image_model.py

```diff
@@ -98,9 +103,15 @@ def preprocess(self, inputs):
             resized_image = pad_image(resized_image, (self.w, self.h))
         resized_image = self.input_transform(resized_image)
         resized_image = self._change_layout(resized_image)
+        resized_image = self.int2float(resized_image)
         dict_inputs = {self.image_blob_name: resized_image}
         return dict_inputs, meta
 
+    def int2float(self, image):
+        if image.dtype == np.uint8 and self.inputs[self.image_blob_name].precision in ('DT_FLOAT', 'FP32'):
```

Comment:
Added the list of precisions to ModelAdapter. However, it was decided to move data type conversion to adapter side, so it will be implemented in the adapters, if it is needed. So, mapping of precisions can be used for unified logging of input layers precisions info.